### ISSUE TITLE
training the code with pretrained weights

### ISSUE THREAD

**Issue by notorious-eric** [codeslake/IFAN#6]
Hi, I would like to ask about how to load the pretrained weights and start training without interfering other process?

**Comment by codeslake** [comment]
Hi, are you trying to resume training? or to start the training with pretrained weights with a new learning rate scheduling?

**Comment by notorious-eric** [comment]
start the training with pretrained weights with a new learning rate scheduling

**Comment by codeslake** [comment]
I see. The current code does not provide the functionality at the moment. I will spend some of my time adding the functionality.
Do your pretrained weights include weights for the reblurring network also?

**Comment by notorious-eric** [comment]
I think so. And i have encountered a warning while training. It said "Warning! No positional inputs found for a module, assuming batch size is 1." But I have set the batchsize as your training details. I wonder what happened?

![image](https://user-images.githubusercontent.com/44058627/159426659-23d254e5-d80f-4227-96b8-3a9f54e18a1a.png)

**Comment by codeslake** [comment]
Hi, I've just pushed some commits. Pull the repo and try the following command:
 * Use a `mode` name different from the `mode` name of the pretrained model to prevent the process from overriding the folder of the pretrained model.
 * Give absolute path of the pretrained weights to `-ra` or `--resume_absolute` option.
 * The learning rate scheduleling will start from the beginning following the config file given with the `--config` option.
```bash
CUDA_VISIBLE_DEVICES=0,1,2,3 python -B -m torch.distributed.launch --nproc_per_node=4 --master_port=9000 run.py \
            --is_train \
            --mode IFAN_finetune \
            --config config_IFAN \
            --trainer trainer \
            --network IFAN \
            -b 2 \
            -th 8 \
            -ra /path/to/the/pretriained/weights \
            -ss \
            -dist
```

**Comment by notorious-eric** [comment]
Thank you for your help! I will try it now.

**Comment by makeyyp** [comment]
I am a beginner and when replicating your code this error occurred.The dataset was placed as required。Can you help me
![image](https://github.com/codeslake/IFAN/assets/137139484/a8bbd5f6-6184-49a5-a4fb-3c53fba66f06)
